Volume/Views/Onboarding/WelcomeView.swift

```diff
@@ -23,7 +23,7 @@ extension OnboardingView {
                     textRegular: "trending in the Cornell commnity and from publications you follow"
                 )
                 FeatureRow(
-                    image: "shout-out",
+                    image: "shout-out-orange",
```

Comment:
I would call this `shout-out` since in theory it can have any color now that it's a template image.